src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -261,7 +272,8 @@
 
   :put!
   (fn [context]
-    (let [desired-state (get-desired-state context)]
+    (let [desired-state (get-desired-state context)
+          settings (merge-request-settings settings context)]
```

Comment:
it might be better to call this `settings'` and pass it in to set-certificate-status below.

Reply:
Agreed, I actually had to go hunting to find out what `settings` was before. This would make it clearer.

Reply:
Moved it out of the let and into the function itself.